ec2/sc-ec2-linux-jumpcloud-workflows-v1.0.0.yaml

```diff
@@ -362,16 +362,26 @@ Resources:
             'Fn::Sub': '${AWS::Region}-sc-enduser-iam-ServiceCatalogEndusers-RoleId'
 Outputs:
   LinuxInstanceId:
+    Description: 'The ID of the EC2 instance'
     Value: !Ref 'LinuxInstance'
   LinuxInstancePrivateIpAddress:
+    Description: 'The IP Address of the EC2 instance'
     Value: !GetAtt 'LinuxInstance.PrivateIp'
   EC2InstanceType:
+    Description: 'The EC2 instance type'
     Value: !Ref 'EC2InstanceType'
   IAMInstanceRole:
+    Description: 'The IAM Role associated with the instance'
     Value: !Ref 'InstanceRole'
   IAMInstanceProfile:
+    Description: 'The IAM Instance Profile associated with the instance'
     Value: !Ref 'InstanceProfile'
   ConnectionInstructions:
+    Description: 'Guidelines on connecting to instances'
```

Comment:
we can probably remove ConnectionInstructions from this one since ConnectionURI is enough?